### PR TITLE
Add hideBackButton prop @0.x

### PR DESCRIPTION
**Pull request by bivillar**
#### What is the purpose of this pull request?
Add `hideBackButton` prop to enable hiding the back button when needed.

#### What problem is this solving?
Enable hiding the button at the profile page as it is not needed.

#### How should this be manually tested?
https://cobasi.myvtex.com/_secure/account?workspace=profile#/profile

#### Screenshots or example usage
Before:
image

After:
image

#### Types of changes

- [ ] Bug fix (a non-breaking change which fixes an issue)
- [x] New feature (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: react/components/ContentWrapper.js [hunk 1]
```diff
@@ -30,24 +30,26 @@ class ContentWrapper extends Component {
...
       headerContent,
       namespace,
+      hideBackButton = false,
     } = this.props
     const { shouldShowError } = this.state
 
+    const backButtonConfigs = {
+      linkLabel: backButton
+        ? backButton.title
+          ? backButton.title
```
Comment (kevinch) [inline react/components/ContentWrapper.js:40]: Is that a typo? Looks like you're checking the same think twice.

File: react/components/ContentWrapper.js [hunk 2]
```diff
@@ -30,24 +30,26 @@ class ContentWrapper extends Component {
...
+          : intl.formatMessage({ id: backButton.titleId })
+        : intl.formatMessage({ id: 'commons.back' }),
+      onLinkClick: () => history.push(backButton ? backButton.path : '/'),
+    }
+
     return (
       <section className="vtex-account__page w-100 w-80-m">
         <header>
```
Comment (kevinch) [inline react/components/ContentWrapper.js:48]: Not related but shouldn't the `header` tag be within the `PageHeader` component?

File: react/components/ContentWrapper.js (hunk 1 above)
Comment (bivillar) [reply react/components/ContentWrapper.js:40]: It's not! 
this translates to 
```
let linkLabel
if (backButton)
    linkLabel = backButton.title ? backButton.title : intl.formatMessage({ id: backButton.titleId })
else 
    linkLabel = intl.formatMessage({ id: 'commons.back' })
```

File: react/components/ContentWrapper.js (hunk 2 above)
Comment (bivillar) [reply react/components/ContentWrapper.js:48]: PageHeader is a component from Styleguide that is used here at the ContentWrapper as the header of the my-account pages!

Comment (kevinch) [reply react/components/ContentWrapper.js:48]: Good point.

**Pull request merged**